packages/appstore/src/components/containers/listing-container.tsx

```diff
@@ -36,9 +35,8 @@ const Options = observer(({ title, description, is_deriv_platform }: TOptionsPro
     return <TitleCardLoader />;
 });
 
-const Switcher = ({ wallet_account, is_deriv_platform }: TSwitcherProps) => {
+const Switcher = ({ is_deriv_platform }: TSwitcherProps) => {
     if (!is_deriv_platform) return null;
-    if (wallet_account) return <WalletTransferBlock wallet_account={wallet_account} />;
```

Comment:
why are we removing this `WalletTransferBlock`? Is it not required?

Reply:
@shahzaib-deriv yeah, you are right, I deleted this yesterday in the 3rd PR of TH performance, so, don't worry, it will be handled. Also, Nijil said to me his team will delete all unused wallet code from appstore package when they finish with all 3 wallet migration steps. For now only 1st step completed